### PR TITLE
solved the issue Python:NumPy Math Methods exp #4721

### LINKED ISSUE #4721: [Term Entry] Python:NumPy Math Methods exp
### Reason/inspiration (optional)

"We would like a new term entry on the `Math Methods` term for [Numpy](https://www.codecademy.com/resources/docs/numpy): exp. The entry should go in a new file under `docs/content/numpy/concepts/math-methods/terms/exp/exp.md`.

The entry should describe the term and how it is used. Also include:
- An "Example" section that explains readers how to use the exp method on a given array.
- Should contain informations for functions and syntax.
- Should have a python codebyte with working code snippet showcasing this function.
Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue."

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by AlinaStepanov**
### Description

Added new file exp.md with explanation of `exp` function, example  and code.

### Issue Solved

Closes #4721

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new-branch after the changes I'm making.
- [ ] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/numpy/concepts/math-methods/terms/exp/exp.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# `exp`
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/exp/exp.md:1]:
```suggestion
---
Title: '.exp()'
Description: 'Computes the exponential of all elements in the input array'
Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:
  - 'Arrays'
  - 'Functions'
  - 'NumPy'
CatalogContent:
  - 'learn-python-3'
  - 'paths/data-science'
---
```

File: content/numpy/concepts/math-methods/terms/exp/exp.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+## Introduction
+
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/exp/exp.md:4]:
```suggestion
```

File: content/numpy/concepts/math-methods/terms/exp/exp.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+The `exp` function in NumPy is used to calculate the exponential of all elements in the input array. This function is a part of the NumPy library, which is widely used for numerical computing in Python. The `exp` function is particularly useful in scientific computations where exponential functions are common.
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/exp/exp.md:5]:
```suggestion
The **`.exp`** function in NumPy calculates the exponential of all elements in the input array. As a part of the NumPy library, which is widely used for numerical computing in Python, this function is particularly useful in scientific computations where exponential functions are common.
```

File: content/numpy/concepts/math-methods/terms/exp/exp.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+## Syntax
+
+```python
+numpy.exp(arr)
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/exp/exp.md:10]:
```suggestion
numpy.exp(arr, out = None, where = True, casting = ‘same_kind’, order = ‘K’, dtype = None)
```

File: content/numpy/concepts/math-methods/terms/exp/exp.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+```
+
+- `arr`: Input array. The elements of the array can be any real numbers.
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/exp/exp.md:13]: Please also add the description of the other parameters as well.

File: content/numpy/concepts/math-methods/terms/exp/exp.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+### Explanation
+
+The `exp` function computes the exponential of each element in the input array `arr`. The exponential function, \(\exp(x)\), returns \(e^x\), where \(e\) is Euler's number (approximately 2.71828).
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/exp/exp.md:17]:
```suggestion
>**Note:** The `.exp()` function computes the exponential of each element in the input array array. The exponential function, `.exp(𝑥)`, returns `e^x` , where `𝑒` is Euler's number, approximately 2.71828. 
```

File: content/numpy/concepts/math-methods/terms/exp/exp.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+
+## Example
+
+Here is an example demonstrating the use of the `exp` method:
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/exp/exp.md:21]:
```suggestion
Here is an example demonstrating the use of the `.exp()` function:
```

File: content/numpy/concepts/math-methods/terms/exp/exp.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+arr = np.array([0, 1, 2, 3])
+
+# Calculate the exponential of each element in the array
+result = np.exp(arr)
+
+# Print the result
+print("Exponential of input array:", result)
+```
+
+### Output
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/exp/exp.md:36]:
```suggestion
The code above produces the following output:
```

File: content/numpy/concepts/math-methods/terms/exp/exp.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+Here is an example demonstrating the use of the `exp` method:
+
+```python
+import numpy as np
+
+# Create an input array
+arr = np.array([0, 1, 2, 3])
+
+# Calculate the exponential of each element in the array
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/exp/exp.md:29]:
```suggestion
# Compute the exponential of each element in the array
```

File: content/numpy/concepts/math-methods/terms/exp/exp.md [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+
+### Output
+
+```
+Exponential of input array: [  1.           2.71828183   7.3890561   20.08553692]
+```
+
+In this example, the `exp` function is used to compute the exponential of each element in the array `[0, 1, 2, 3]`. The output shows the result of applying the exponential function to each element of the array.
+
+## Python Codebyte
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/exp/exp.md:44]:
```suggestion
## Codebyte Example
```

File: content/numpy/concepts/math-methods/terms/exp/exp.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
+# `exp`
+
+## Introduction
+
+The `exp` function in NumPy is used to calculate the exponential of all elements in the input array. This function is a part of the NumPy library, which is widely used for numerical computing in Python. The `exp` function is particularly useful in scientific computations where exponential functions are common.
+
+## Syntax
+
+```python
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/exp/exp.md:9]:
```suggestion
```pseudo
```

File: content/numpy/concepts/math-methods/terms/exp/exp.md [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+
+### Explanation
+
+The `exp` function computes the exponential of each element in the input array `arr`. The exponential function, \(\exp(x)\), returns \(e^x\), where \(e\) is Euler's number (approximately 2.71828).
+
+## Example
+
+Here is an example demonstrating the use of the `exp` method:
+
+```python
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/exp/exp.md:23]:
```suggestion
```py
```

File: content/numpy/concepts/math-methods/terms/exp/exp.md [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+Exponential of input array: [  1.           2.71828183   7.3890561   20.08553692]
+```
+
+In this example, the `exp` function is used to compute the exponential of each element in the array `[0, 1, 2, 3]`. The output shows the result of applying the exponential function to each element of the array.
+
+## Python Codebyte
+
+Here is a working code snippet demonstrating the `exp` function in action:
+
+```python
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/exp/exp.md:48]:
```suggestion
```codebyte/python
```

File: content/numpy/concepts/math-methods/terms/exp/exp.md [hunk 14]
```diff
@@ -0,0 +1,68 @@
...
+import numpy as np
+
+# Create an input array
+arr = np.array([0, 1, 2, 3])
+
+# Calculate the exponential of each element in the array
+result = np.exp(arr)
+
+# Print the result
+print("Exponential of input array:", result)
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/exp/exp.md:58]: Please change the codebyte example, the example and codebyte example can't be the same. Use a different code (or values) for the codebyte example.

File: content/numpy/concepts/math-methods/terms/exp/exp.md [hunk 15]
```diff
@@ -0,0 +1,68 @@
...
+```
+
+### Explanation of Codebyte
+
+1. **Import NumPy**: The code begins by importing the NumPy library, which is essential for numerical operations in Python.
+2. **Create an Array**: An array `arr` is created with the values `[0, 1, 2, 3]`.
+3. **Calculate Exponential**: The `np.exp` function is applied to the array `arr`, calculating the exponential of each element.
+4. **Print Result**: The result is printed, showing the exponential values of the input array elements.
+
+By following these steps, you can easily use the `exp` function in NumPy to perform exponential calculations on arrays.
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/exp/exp.md:68]:
```suggestion
```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @AlinaStepanov! Thank you for contributing to the Codecademy Docs! The entry was well written! 😄 

I've suggested a few changes for this entry, could you please review and modify those at your earliest convenience? Thank you! 😃

**Review by mamtawardhani — APPROVED** [review]
@AlinaStepanov Thank you for making the changes! Looks good for a second review! 😄

**Review by Sriparno08 — APPROVED** [review]
Looks good, @AlinaStepanov! 👍

**Pull request merged**